README.md

```diff
@@ -1 +1,70 @@
-# Hello-world-project
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello World
+
+> Description the project.
```

Comment:
- [ ] I kindly suggest that you replace this placeholder with the description of your project.

## EXAMPLE
`This is a simple Hello Microverse project built with HTML & CSS`